### PR TITLE
Updates for handling breaking changes introduced by OpenZeppelin.

### PR DESCRIPTION
**Pull request by zeuslawyer**
OZ released v4.6.0 on 26 April 2022 and it has breaking changes [1].

This PR
- Updates GovernorContract, GovernanceTokenWrapper.

Note that as of today the Open Zeppelin Wizard's auto-generated
code does not reflect the breaking changes.

TESTS: ran `yarn hardhat test --deploy-fixture`. The flag wasn't necessary but added it
as some people may need it depending on their config.

[1] https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/CHANGELOG.md#breaking-changes

### CODE REVIEW

File: hardhat.config.ts [hunk 1]
```diff
@@ -6,7 +6,6 @@ import "hardhat-gas-reporter"
 import "dotenv/config"
 import "solidity-coverage"
 import "hardhat-deploy"
-import "solidity-coverage"
```
Comment (PatrickAlphaC) [inline hardhat.config.ts:9]: Why is this removed?

File: scripts/queue-and-execute.ts [hunk 2]
```diff
@@ -36,7 +36,7 @@ export async function queueAndExecute() {
     descriptionHash
   )
   await executeTx.wait(1)
-  console.log(await box.retrieve().toString())
+  console.log(`Box value: ${await box.retrieve()}`)
```
Comment (PatrickAlphaC) [inline scripts/queue-and-execute.ts:39]:
```suggestion
  console.log(`Box value: ${await box.retrieve().toString()}`)
```

**Review by PatrickAlphaC — COMMENTED** [review]
Thank you!! Could you take a look at my comments?

File: scripts/queue-and-execute.ts (hunk 2 above)
Comment (zeuslawyer) [reply scripts/queue-and-execute.ts:39]: It was printing out [Object object] with `toString()`.  Not sure if this is newer node runtime behaviour with object.prototype.toString() - but seems to date back to 2017 [see first reply in 1] and see below screenshot of output.

Screen Shot 2022-05-08 at 7 02 28 pm

By removing `toString()` I got it to provide the returned number.  JS magic?!

Happy to leave as is  -your call! 


[1] https://stackoverflow.com/questions/46146860/javascript-template-strings-dont-pretty-print-objects

File: hardhat.config.ts (hunk 1 above)
Comment (zeuslawyer) [reply hardhat.config.ts:9]: Since Hardhat does some magical woowoo by injecting imports into the global scope (which can "pollute" global scope), I didnt find anything that was actually using this package in the scripts and tests.  Happy to reinstate, but since HH injects all sorts o f things I felt maybe removing it is one less then for your users to analyse /evaluate when studying this config file.   So purely hygiene -- but on the basis that this package does not "appear" to be used.  Is it?

Comment (PatrickAlphaC) [reply hardhat.config.ts:9]: No I think you're right.

File: scripts/queue-and-execute.ts (hunk 2 above)
Comment (PatrickAlphaC) [reply scripts/queue-and-execute.ts:39]: Yep, I'm being dumb. Thank you!

**Comment by PatrickAlphaC** [comment]
Boom! Thanks @zeuslawyer !!!
